shared/media-accelerator/get-started/_get-started-sdk.mdx

```diff
@@ -0,0 +1,168 @@
+
+<ProjectTest />
+
+## Reference
+
+This section contains information that completes the information in this page, or points you to documentation that explains other aspects to this product.
+
+### Intercommunication with RTC SDKs
+
+If your business scenarios require not only high-quality audio and video transmission, but also easy integration, low memory usage, and low power consumption, such as smart cameras and smart doorbells, Agora recommends that you use the RTC SDK (v4.x) with the <Vpd k="SDK" />.
+
+The following table shows the data formats supported when the <Vpd k="SDK" /> communicates with the RTC SDK (v4.x):
+
+| SDK Type | <Vpd k="SDK"/> |
+|----|---------------|
+| RTC Native/third-party framework SDK (v4.x) | <ul><li>Audio: G722, G711, Opus, AAC</li><li>Video: H.264, JPEG</li></ul> |
+| RTC Web SDK (v4.x) | <ul><li>Audio: G722, G711, Opus</li><li>Video: H.264</li></ul> |
+| RTC applet SDK | <ul><li>Audio: G711, Opus</li><li>Video: H.264, JPEG</li></ul> |
+
+The sending and receiving ends can realize intercommunication only if the set data format is consistent and complies with the above table.
+
+The following sections describe how to set the audio and video data format in the SDK.
+
+#### Setup <Vpd k="SDK" />
+
+**Audio**
+
+<Vpd k="SDK" /> only supports mono audio data, and the data sending interval is 20 ms.
+
+* To send and receive AAC audio data, the operation is as follows: 
+
+    When calling the `agora_rtc_send_audio_data` method to send audio data, set `audio_frame_info_t.data_typeto` `AUDIO_DATA_TYPE_AACLC` or `AUDIO_DATA_TYPE_HEAAC` data type, and the corresponding data size. The size of AAC audio data needs to be resolved by the user.
+
+* To send and receive Opus, G722, G711 audio data, the operation is as follows:
+
+    1. When calling the `agora_rtc_join_channel` method to join the RTC channel, set `rtc_channel_options_t.audio_codec_opt.audio_codec_type` to `AUDIO_CODEC_TYPE_OPUS`, `AUDIO_CODEC_TYPE_G722`, `AUDIO_CODEC_TYPE_G711A` or `AUDIO_CODEC_TYPE_G711U` encoding format, and the corresponding sampling rate.
+
+    1. When calling the `agora_rtc_send_audio_data` method to send audio data, set `audio_frame_info_t.data_type` to `AUDIO_DATA_TYPE_PCM` and the corresponding data size.
+
+The table below shows the recommended sampling rates and corresponding data sizes for Opus, G722, and G711 audio data.
+
+| Audio format | Sampling rate (Hz) | Audio data size (Byte) |
+|------|------|-------|
+| G711 | 8000 | 320 |
+| G722 | 16000 | 640 |
+| OPUS | 16000 | 640 |
+| OPUS | 48000 | 1920 |
+
+**Video**
+
+In <Vpd k="SDK" />, H.264 video data is sent and received by default.
+
+To send and receive JPEG video data, when calling the `agora_rtc_send_video_data` method to send video data, set `video_frame_info_t.data_type` to `VIDEO_DATA_TYPE_GENERIC_JPEG`.
+
+#### Set up RTC Native/third-party framework SDK (v4.x)
+
+**Audio**
```

Comment:
Again, I don't get why we have all this here rather than in a doc explaining how it works, plus API ref.

Reply:
This is stuff you would need if you want your RTSA SDK app to talk to an RTC SDK app with some special audio/video settings.
Moving some relevant stuff to the call quality doc.
I will add the rest as a new doc titled "Intercommunication with RTC SDKs" under Reference docs.